apps/01PigLatin.js

```diff
@@ -6,8 +6,87 @@ prompt.start();
 
 
 function pigLatin(word) {
+  // Concept Check
+ //
+ // Please explain why we are using String.prototype.toLowerCase() here.
+ // For more info:
+ // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+ //
+ // Your Answer:
+ //replaces any upper case letter with lowercase so less lines are required to accomodate them.
+ word  = word.toLowerCase();
```

Comment:
The answer here is more like, we change the word to lowercase so that we only have to check lower case vowels.

Keep your answers conceptual here.

![finn and jake](https://media.giphy.com/media/HozURFRL6mT2o/giphy.gif)